pyiron/lammps/control.py

```diff
@@ -641,13 +641,14 @@ def calc_vcsgc(
             append_if_not_present=True
         )
 
-    def measure_mean_value(self, key, every=1, repeat=None, freq=None):
+    def measure_mean_value(self, key, every=1, repeat=None, freq=None, name=None):
         """
             Args:
```

Comment:
Please add the first comment + first sentence of your second comment from the 'conversation' section of the pull request to the docstring. They might need light formatting, but I think they really clearly communicate the intention of this function.